pandas/tseries/tests/test_timeseries.py

```diff
@@ -3291,6 +3291,12 @@ def test_NaT_scalar(self):
         series[2] = val
         self.assertTrue(com.isnull(series[2]))
 
+    def test_NaT_cast(self):
+        # GH10747
+        result = Series([iNaT]).astype('datetime64[ms]')
```

Comment:
ah you mean the fact that the `astype` to `datetime[ns]` works on master is a bug?